src/components/ebay-textbox/template.marko

```diff
@@ -1,4 +1,81 @@
+<if(data.initIcons)>
+<span style="display: none">
```

Comment:
This line should be tabbed (4 spaces) at the beginning of the line. (Whitespace is off.)

Reply:
Not sure why I didn't get a lint error. This will be fixed in my next commit.